nursery/decode-data-using-url-encoding.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: decode data using URL encoding
```

Comment:
`decode URL-encoded data`?

Reply:
This was my initial choice but it does not adhere to the naming scheme we have been using e.g.
- `decode data using Base64 in .NET`
- `decrypt data using RSA`
- `decrypt data via SSPI`

Happy to make the suggested change pending additional thoughts

Reply:
Hm, right, I'm fine leaving as is then.